integration/combination/test_function_with_implicit_api_with_timeout.py

```diff
@@ -0,0 +1,11 @@
+from unittest.case import skipIf
+
+from integration.config.service_names import REST_API
+from integration.helpers.base_test import BaseTest
+from integration.helpers.resource import current_region_does_not_support
+
+
+@skipIf(current_region_does_not_support([REST_API]), "REST API is not supported in this testing region")
+class TestFunctionWithImplicitApiWithTimeout(BaseTest):
+    def test_function_with_implicit_api_with_timeout(self):
+        self.create_and_verify_stack("combination/function_with_implicit_api_with_timeout")
```

Comment:
We're only checking if the stack deploys here. Could we add some code to check if the property is working as expected? Maybe something [like this](https://github.com/aws/serverless-application-model/blob/develop/integration/combination/test_http_api_with_cors.py#L40-L48)

Reply:
yep good idea